src/test/java/seedu/address/logic/commands/DeleteTaskCommandTest.java

```diff
@@ -36,13 +36,15 @@
     @Test
     public void constructor_nullArgs_throwsNullPointerException() {
         Index targetTasksIndex = Index.fromZeroBased(model.getFilteredPersonList()
-                .get(1)
-                .getTasks()
-                .size());
+                        .get(1)
```

Comment:
This seems to be too much indentation, not sure if its a display error on my side.